aten/src/ATen/native/mkldnn/Conv.cpp

```diff
@@ -66,28 +63,17 @@ ideep::tensor _mkldnn_conv2d(
     at::IntArrayRef stride,
     at::IntArrayRef dilation,
     int64_t groups) {
-  std::vector<int64_t> kernel_size(x.ndims());
-  // mkldnn conv2d weights could have been re-ordered to 5d by
-  // mkldnn_reorder_conv2d_weight
-  if (w.ndims() == x.ndims() + 1) {
-    AT_ASSERTM(
-        groups > 1,
-        "Only group _mkldnn_conv2d weights could have been reordered to 5d");
-    kernel_size[0] = w.get_dim(0) * w.get_dim(1);
-    std::copy_n(
-        w.get_dims().cbegin() + 2, x.ndims() - 1, kernel_size.begin() + 1);
-  } else {
-    std::copy_n(w.get_dims().cbegin(), x.ndims(), kernel_size.begin());
-  }
 
-  const ideep::param::dims x_dims = x.get_dims();
+  auto kernel_size = w.get_dims();
+
+  auto x_dims = x.get_dims();
   std::vector<int64_t> input_size{x_dims.cbegin(), x_dims.cend()};
```

Comment:
Can we save the vector construction here given x_dims is int64 now?

Reply:
yep